manager-rs/dauth-service/src/data/context.rs

```diff
@@ -23,9 +20,8 @@ pub struct DauthContext {
 
 #[derive(Debug)]
 pub struct LocalContext {
-    pub database: Mutex<HashMap<Id, VecDeque<AuthVectorRes>>>,
-    pub kseaf_map: Mutex<HashMap<HresStar, Kseaf>>,
-    pub user_info_database: Mutex<HashMap<Id, UserInfo>>,
+    pub database_path: String,
```

Comment:
Is the database path string used? It seems like once the pool is created we don't need to keep the path around during runtime?

Reply:
Good point! I'll remove it.